test/tracing/mzcompose.py

```diff
@@ -0,0 +1,100 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+# Tests the dynamic tracing setup on environmentd
```

Comment:
Can you make this a documentation comment in the module instead? Then it should show up in `./mzcompose list` or whatever the command is.

Reply:
oh lmao i forgot about that in python lol

Reply:
I think only function-level ones show up so ill add it to `workflow_default`